<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FEditServ</class>
 <widget class="QWidget" name="FEditServ">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактирование записи автосбора</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Название группы файлов:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_opis">
       <property name="minimumSize">
        <size>
         <width>381</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Исходная директория:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_isdir">
         <property name="minimumSize">
          <size>
           <width>361</width>
           <height>32</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_isdir">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/res/1/folder.png</normaloff>:/res/1/folder.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Маска файлов для обработки:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_maska">
       <property name="minimumSize">
        <size>
         <width>401</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Выходная директория:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_vixdir">
           <property name="minimumSize">
            <size>
             <width>361</width>
             <height>32</height>
            </size>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_vixdir">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/res/1/folder_go.png</normaloff>:/res/1/folder_go.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QCheckBox" name="cb_prvd">
           <property name="text">
            <string>Переименовать по маске:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_maskvd">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Доп.директории</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="rb_ns">
          <property name="text">
           <string>Не создавать</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_dat">
          <property name="text">
           <string>Дата сбора</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_god">
          <property name="text">
           <string>Год, сутки</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_sut">
          <property name="text">
           <string>Дата информации</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Директория для копирования:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_copdir">
           <property name="minimumSize">
            <size>
             <width>331</width>
             <height>32</height>
            </size>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_copdir">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/res/1/folder_go.png</normaloff>:/res/1/folder_go.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QCheckBox" name="cb_prcd">
           <property name="text">
            <string>Переименовать по маске:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_maskcd">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Доп.директории</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QRadioButton" name="rb_nsc">
          <property name="text">
           <string>Не создавать</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_datc">
          <property name="text">
           <string>Дата сбора</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_godc">
          <property name="text">
           <string>Год, сутки</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_sutc">
          <property name="text">
           <string>Дата информации</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="cb_copdobr">
         <property name="text">
          <string>Скопировать до обработки</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cb_coppobr">
         <property name="text">
          <string>Скопировать после обработки</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cb_delisx">
         <property name="text">
          <string>Удалить исходный</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="cb_rarx">
         <property name="text">
          <string>Распаковать архив</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cb_rhata">
         <property name="text">
          <string>Распаковать Хатанака</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cb_uhata">
         <property name="text">
          <string>Упаковать Хатанака</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cb_uarx">
         <property name="text">
          <string>Упаковать архив</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_14">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QPushButton" name="pbSave">
         <property name="text">
          <string>Сохранить</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/res/1/accept_button.png</normaloff>:/res/1/accept_button.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbCancel">
         <property name="text">
          <string>Отменить</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/res/1/cancel.png</normaloff>:/res/1/cancel.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>365</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
